addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Maintenance changes.
 - Rename of Feature-Policy header to Permissions-Policy to follow spec change.
 - Update links to repository.
+- Ignore CSS or JavaScript files when scanning for source code disclosures (Issue 6595).
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
Done. Moved to unreleased section.